pilot/pkg/config/kube/gateway/testdata/mesh.yaml

```diff
@@ -78,3 +78,38 @@ spec:
     backendRefs:
     - name: echo
       port: 80
+---
+apiVersion: gateway.networking.k8s.io/v1alpha2
+kind: HTTPRoute
+metadata:
+  name: echo-port
+  namespace: default
+spec:
+  parentRefs:
+  - kind: Service
+    name: echo-port
+    port: 80
```

Comment:
Are we required to support section too ? Port name ?

Reply:
I don't think so.. let me double check the spec